stubs/fortify.php

```diff
@@ -74,7 +73,7 @@
     |
     */
 
-    'home' => RouteServiceProvider::HOME,
+    'home' => '/home',
```

Comment:
I think we should unify on `/dashboard` here, which is what Jetstream uses.

Reply:
@taylorotwell I apply that change via Jetstream's install command.